zerver/tests/test_messages.py

```diff
@@ -4132,7 +4129,6 @@ def test_maybe_update_first_visible_message_id(self) -> None:
 
         RealmCount.objects.create(realm=realm, property=stat.property,
                                   end_time=end_time, value=5)
-        with mock.patch('zerver.lib.message.cache_get', return_value=True), \
```

Comment:
`cache_get` is just an unused import inside `zerver.lib.message`. Is this a (preexisting) bug?